napari/_qt/menus/_util.py

```diff
@@ -101,3 +101,22 @@ def populate_menu(menu: QMenu, actions: List['MenuItem']):
                 @emitter.connect
                 def _setchecked(e, action=action):
                     action.setChecked(e.value if hasattr(e, 'value') else e)
+
+
+def populate_qmenu_from_manifest(menu: QMenu, menu_key: str):
```

Comment:
note: I removed the "test menu" from this PR.  we can add specific menu targets later